magic_items.py

```diff
@@ -0,0 +1,387 @@
+            powers_out += f'\t\t\t\t\t<recharge type="string">{power["recharge"]}</recharge>\n'
+        if power["shortdescription"] != '':
+            powers_out += f'\t\t\t\t\t<shortdescription type="string">{power["shortdescription"]}</shortdescription>\n'
+        powers_out += f'\t\t\t\t</id-{entrry_id}>\n'
+    powers_out += '\t\t\t'
+    return re.sub('^\s*$', '', powers_out)
+
+def props_format(props_in):
+    id = 0
+
+    # Split the input at each \n into a list of properties
+    props_list = re.split(r'\\n', props_in)
+
+    # Loop though properties list to create all the tags
+    props_out = '\n'
+    for p in props_list:
+        id += 1
+        entrry_id = "00000"[0:len("00000")-len(str(id))] + str(id)
+        props_out += f'\t\t\t\t<id-{entrry_id}>\n'
+        props_out += f'\t\t\t\t\t<shortdescription type="string">{p}</shortdescription>\n'
+        props_out += f'\t\t\t\t</id-{entrry_id}>\n'
+    props_out += '\t\t\t'
+    return re.sub('^\s*$', '', props_out)
+
+if __name__ == '__main__':
+
+    alt_reward_expr = re.compile('(Alternative Reward|Battle Scars|Divine Boon|Echo of Power|Elemental gift|Elemental Gift|Fey Magic Gift|Glory Boon|Grandmaster Training|Legendary Boon|Lost Rune|Primal Blessing|Psionic Talent|Secret of the Way|Sorcerer-King\'s Boon|Templar Brand|Veiled Alliance Mystery|Wanderer\'s Secret)')
+
+    # Pull data from Portable Compendium
+    db = []
+    try:
+        db = create_db('ddiItem.sql', "','")
+    except:
+        print("Error reading data source.")
+
+    print(f"{len(db)} entries recovered")
+
+    # Convert data to FG format
+    print("converting to FG format...")
+
+    # Initialize all modules databases
+    export_list = []
+
+    if not db:
+        print("NO DATA FOUND IN SOURCES, MAKE SURE YOU HAVE COPIED YOUR 4E PORTABLE COMPENDIUM DATA TO SOURCES!")
+        input('Press enter to close.')
+        sys.exit(0)
+
+    for i, row in enumerate(db):
+
+        # Parse the HTML text 
+        html = row['Txt']
+        html = html.replace('\\r\\n','\r\n').replace('\\','')
+        parsed_html = BeautifulSoup(html, features="html.parser")
+
+        # Retrieve the data with dedicated columns
+        name_str =  row['Name'].replace('\\', '')
+        category_str = row['Category'].replace('\\', '')
+        rarity_str =  row['Rarity'].replace('\\', '')
+
+        bonus_str = ''
+        class_str = ''
+        cost_str = ''
+        critical_str = ''
+        enhancement_str = ''
+        flavor_str = ''
+        group_str = ''
+        level_str = ''
+        mitype_str = ''
+        powers_str = ''
+        prerequisite_str = ''
+        properties_str = ''
+        props_str = ''
+        special_str = ''
+        subclass_str = ''
+
+        # Class
+        if (re.search('^(Alchemical Item|Alternative Reward|Ammunition|Arms|Companion|Consumable|Familiar|Feet|Head|Head and Neck|Neck|Mount|Ring|Waist|Wondrous)$', category_str) and re.search('^(Common|Uncommon|Rare)', rarity_str)):
```

Comment:
It looks like "Hands" is missing from this list? Otherwise everything else is covered!